src/components/AppRoot/AppRoot.tsx

```diff
@@ -52,13 +57,14 @@ export const AppRoot = withAdaptivity<AppRootProps>(
     hasMouse,
     noLegacyClasses = false,
     scroll = "global",
+    portalRoot,
```

Comment:
```suggestion
    portalRoot: portalRootProp
```
по классике (как в других местах имею ввиду) стоит обозвать так и отказаться от префикса `_`

Reply:
done